.travis.yml

```diff
@@ -17,21 +17,22 @@ matrix:
       env:
         - CS_CHECK=true
     - php: 7.1
-    - php: hhvm 
+    - php: hhvm
   allow_failures:
     - php: hhvm
 
 before_install:
   - if [[ $TEST_COVERAGE != 'true' ]]; then phpenv config-rm xdebug.ini || return 0 ; fi
   - composer self-update
   - if [[ $TEST_COVERAGE == 'true' ]]; then composer require --dev --no-update satooshi/php-coveralls:^1.0 ; fi
+  - if [[ $TEST_COVERAGE == 'true' ]]; then composer require --dev --no-update michaelmoussa/php-coverage-checker:^1.1.0 ; fi
 
 install:
   - travis_retry composer install --no-interaction
   - composer info -i
 
 script:
-  - if [[ $TEST_COVERAGE == 'true' ]]; then composer test-coverage ; else composer test ; fi
+  - if [[ $TEST_COVERAGE == 'true' ]]; then composer test-coverage && vendor/bin/php-coverage-checker clover.xml 100 ; else composer test ; fi
```

Comment:
Yes, but I'd need to add my coverage checker library to the dev dependencies. I did it this way in order to not make the project itself require a personal library for something that only matters during CI (similar to how it's done for Coveralls now).

Alternatively, I could do something like:

```json
"scripts": {
    ...,
    "check-coverage": "vendor/bin/php-coverage-checker clover.xml 100"
}
```

and...

```yaml
  - if [[ $TEST_COVERAGE == 'true' ]]; then composer test-coverage ; else composer test ; fi
  - if [[ $TEST_COVERAGE == 'true' ]]; then composer check-coverage ; fi
```